info.json

```diff
@@ -0,0 +1,26 @@
+{
+    "name": "connectorHealthMonitoring",
+    "version": "1.0.0",
+    "type": "solutionpack",
+    "local": true,
+    "label": "Connector Health Monitoring",
+    "dependencies": [],
+    "prerequisite": null,
+    "publisher": "Fortinet",
+    "certified": "true",
+    "description": "Solution Pack send the connector health status notification via email based on provided connector list.",
```

Comment:
@cs-abhishek-shukla , please review this and suggest changes, if required

Reply:
```     "description": "Solution Pack sends connector health status notification via email based on user-specified connectors list.", ```

Reply:
Committed